test/helper-config.ts

```diff
@@ -6,3 +6,5 @@ export const INVALID_EMAIL: string = process.env.INVALID_EMAIL ?? "invalid";
 export const LOGIN_URL: string =
   process.env.LOGIN_URL ?? "http://localhost:8080/account/login";
 export const SIGNUP_URL: string = process.env.SIGNUP_URL ?? " ";
+export const MYBOOKSBEFORELOGIN_URL: string = process.env.MYBOOKSBEFORELOGIN_URL ?? "http://localhost:8080/account/login?redirect=%2Faccount%2Fbooks";
```

Comment:
Would be better to read :smile: 
```suggestion
export const MY_BOOKS_BEFORE_LOGIN_URL: string = process.env.MYBOOKSBEFORELOGIN_URL ?? "http://localhost:8080/account/login?redirect=%2Faccount%2Fbooks";
```